redis/src/aio/multiplexed_connection.rs

```diff
@@ -77,13 +79,15 @@ struct Pipeline<SinkItem> {
     sender: mpsc::Sender<PipelineMessage<SinkItem>>,
 
     push_manager: Arc<ArcSwap<PushManager>>,
+    is_stream_closed: Arc<AtomicBool>,
 }
 
 impl<SinkItem> Clone for Pipeline<SinkItem> {
     fn clone(&self) -> Self {
         Pipeline {
             sender: self.sender.clone(),
             push_manager: self.push_manager.clone(),
+            is_stream_closed: self.is_stream_closed.clone(),
```

Comment:
If all the items are simply being `cloned` by calling `.clone()` I believe you can simply add `#[derive(Clone)]` and remove this `impl<SinkItem> Clone for Pipeline<SinkItem> {` completely

Reply:
yes, good idea, changed